search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -184,8 +190,16 @@ internal fun buildTemporalQuery(params: MultiValueMap<String, String>): Temporal
         if (it >= 1) it else null
     }
 
+    val expandedAttrs = attrsParam
+        ?.split(",")
+        .orEmpty()
+        .map {
+            JsonLdUtils.expandJsonLdKey(it, contextLink)!!
+        }
+        ?.toSet()
+
     return TemporalQuery(
-        attrs = attrsParam?.split(",")?.toSet().orEmpty(),
+        attrs = expandedAttrs,
```

Comment:
for clarity, you could rename this attribute to `expandedAttrs`

Reply:
ok